crates/coresimd/src/lib.rs

```diff
@@ -38,6 +37,10 @@
     powerpc_target_feature,
     wasm_target_feature
 )]
+#![cfg_attr(
```

Comment:
Could you add a comment explaining why unconditionally enabling this fails ?

Reply:
added